spec/support/capybara.rb

```diff
@@ -15,14 +15,7 @@
 Capybara.disable_animation = true
 
 Capybara::Screenshot.autosave_on_failure = true
-
-module Capybara
-  module Screenshot
-    def self.capybara_tmp_path
-      Rails.root.join("tmp", "screenshots#{ENV["GROUPS_UNDERSCORE"]}")
-    end
-  end
-end
+Capybara.save_path = Rails.root.join("tmp", "screenshots#{ENV["GROUPS_UNDERSCORE"]}")
```

Comment:
method redefined (warning). I'm like 90% sure this is equivalent, but may want to check CI artifacts work.